src/u2f/u2f_app.c

```diff
@@ -29,6 +29,102 @@ static const app_t _apps[] = {
+        .app_id = "\xa3\x4d\x30\x9f\xfa\x28\xc1\x24\x14\xb8\xba\x6c\x07\xee\x1e\xfa\xe1\xa8\x5e\x8a"
+                  "\x04\x61\x48\x59\xa6\x7c\x04\x93\xb6\x95\x61\x90",
+        .name = "Bitwarden",
+    },
+    {
+        // sha256('https://www.dashlane.com')
+        .app_id = "\x68\x20\x19\x15\xd7\x4c\xb4\x2a\xf5\xb3\xcc\x5c\x95\xb9\x55\x3e\x3e\x3a\x83\xb4"
+                  "\xd2\xa9\x3b\x45\xfb\xad\xaa\x84\x69\xff\x8e\x6e",
+        .name = "Dashlane",
+    },
+    {
+        // sha256('https://www.dropbox.com/u2f-app-id.json')
+        .app_id = "\xc5\x0f\x8a\x7b\x70\x8e\x92\xf8\x2e\x7a\x50\xe2\xbd\xc5\x5d\x8f\xd9\x1a\x22\xfe"
+                  "\x6b\x29\xc0\xcd\xf7\x80\x55\x30\x84\x2a\xf5\x81",
+        .name = "Dropbox",
+    },
+    {
+        // sha256('facebook.com')
+        .app_id = "\x31\x19\x33\x28\xf8\xe2\x1d\xfb\x6c\x99\xf3\x22\xd2\x2d\x7b\x0b\x50\x87\x78\xe6"
+                  "\x4f\xfb\xba\x86\xe5\x22\x93\x37\x90\x31\xb8\x74",
+        .name = "Facebook",
+    },
+    {
+        // sha256('https://www.fastmail.com')
+        .app_id = "\x69\x66\xab\xe3\x67\x4e\xa2\xf5\x30\x79\xeb\x71\x01\x97\x84\x8c\x9b\xe6\xf3\x63"
+                  "\x99\x2f\xd0\x29\xe9\x89\x84\x47\xcb\x9f\x00\x84",
+        .name = "FastMail",
+    },
+    {
+        // sha256('https://id.fedoraproject.org/u2f-origins.json')
+        .app_id = "\x9d\x61\x44\x2f\x5c\xe1\x33\xbd\x46\x54\x4f\xc4\x2f\x0a\x6d\x54\xc0\xde\xb8\x88"
+                  "\x40\xca\xc2\xb6\xae\xfa\x65\x14\xf8\x93\x49\xe9",
+        .name = "Fedora",
+    },
+    {
+        // sha256('https://account.gandi.net/api/u2f/trusted_facets.json')
+        .app_id = "\xa4\xe2\x2d\xca\xfe\xa7\xe9\x0e\x12\x89\x50\x11\x39\x89\xfc\x45\x97\x8d\xc9\xfb"
+                  "\x87\x76\x75\x60\x51\x6c\x1c\x69\xdf\xdf\xd1\x96",
+        .name = "Gandi",
+    },
+    {
+        // sha256('https://gitlab.com')
+        .app_id = "\xe7\xbe\x96\xa5\x1b\xd0\x19\x2a\x72\x84\x0d\x2e\x59\x09\xf7\x2b\xa8\x2a\x2f\xe9"
+                  "\x3f\xaa\x62\x4f\x03\x39\x6b\x30\xe4\x94\xc8\x04",
+        .name = "GitLab",
+    },
+    {
+        // sha256('https://keepersecurity.com')
+        .app_id = "\x53\xa1\x5b\xa4\x2a\x7c\x03\x25\xb8\xdb\xee\x28\x96\x34\xa4\x8f\x58\xae\xa3\x24"
+                  "\x66\x45\xd5\xff\x41\x8f\x9b\xb8\x81\x98\x85\xa9",
+        .name = "Keeper",
+    },
+    {
+        // sha256('https://slushpool.com/static/security/u2f.json')
+        .app_id = "\x08\xb2\xa3\xd4\x19\x39\xaa\x31\x66\x84\x93\xcb\x36\xcd\xcc\x4f\x16\xc4\xd9\xb4"
+                  "\xc8\x23\x8b\x73\xc2\xf6\x72\xc0\x33\x00\x71\x97",
+        .name = "Slush Pool",
+    },
+    {
+        // sha256('https://dashboard.stripe.com')
+        .app_id = "\x2a\xc6\xad\x09\xa6\xd0\x77\x2c\x44\xda\x73\xa6\x07\x2f\x9d\x24\x0f\xc6\x85\x4a"
+                  "\x70\xd7\x9c\x10\x24\xff\x7c\x75\x59\x59\x32\x92",
+        .name = "Stripe",
+    },
+    {
+        // sha256('https://www.dashlane.com')
```

Comment:
dashlane is present twice, please delete one

Reply:
I think I'm going blind. Fixed :)